src/core_plugins/kibana/common/field_formats/types/_numeral.js

```diff
@@ -1,6 +1,6 @@
 import _ from 'lodash';
-import numeral from 'numeral';
-import { FieldFormat } from 'ui/index_patterns/_field_format/field_format';
+import numeral from '@spalger/numeral';
```

Comment:
I created `@elastic/numeral-js` and upgraded to numeral 3.0 at the same time, pretty sure we'll have to publish the old version at `@spalger/numeral-js` in the new package if we want to move it over. That said, I'm not opposed.